docs/get-started/deployment-and-usage.md

```diff
@@ -162,6 +162,20 @@ nfd-master args, in which case nfd-master verifies that the NodeName presented
 by nfd-worker matches the Common Name (CN) of its certificate. This means that
 each nfd-worker requires a individual node-specific TLS certificate.
 
+#### Automated TLS certificate management using cert-manager
+
+[cert-manager](https://cert-manager.io/) can be used to automate certificate
+management between nfd-master and the nfd-worker pods. The instructions below describe
+steps how to set up cert-manager's [CA Issuer](https://cert-manager.io/docs/configuration/ca/) to
+sign `Certificate` requests for NFD components in `node-feature-discovery` namespace.
+
+1. Deploy cert-manager: `$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.0.3/cert-manager.yaml`
+2. Generate `ca.key` and `ca.crt` and add them to `cert-manager.yaml.template`
```

Comment:
I think we should move the instructions from the template to here. I.e. the openssl examples as a reference. Also, we could have a command line example how to add it correctly to the template, e.g. something like:
```
sed s"/tls.key:.*/tls.key: $(cat ca.key|base64 -w 0)/" -i cert-manager.yaml
sed s"/tls.crt:.*/tls.crt: $(cat ca.crt|base64 -w 0)/" -i cert-manager.yaml
```

Reply:
makes sense